webview/src/test/experimentsTable.tsx

```diff
@@ -104,3 +105,8 @@ export const expandRow = (label: string) => {
 
 export const selectedRows = () =>
   screen.queryAllByRole('row', { selected: true })
+
+export const advanceTimersByTime = (ms: number) =>
+  act(() => {
+    jest.advanceTimersByTime(ms)
```

Comment:
`jest` has been removed everywhere in the other file. Should it be the same here?

Reply:
I replaced the call in the other file with this function  so that jest.advaceTimersByTime is always wrapped in act. I kept the name to like the behaviour. Should I change it to make it more obvious why we need it?

Reply:
Sorry, I somehow thought that some of them were imported from jest anyway. (First PR to review on a Monday morning). Just nevermind the comment.